app/models/application_record.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
+  primary_abstract_class
```

Comment:
- [ ] Good job with your API as it works as expected. But I noticed that there are still some things you should fix in your **[frontend](https://github.com/alanluqman/hello-react-front-end/pull/1)** pull request in order for your project to be completed. Congratulations :clap:

Reply:
thanks for your review dear @tresorsawasawa , i will do.